web-server/assets/src/script/skills.js

```diff
@@ -76,7 +76,24 @@ const filterSkillsList = filter => {
   })
 }
 
+const hideEmptyFilters = (filterElements) => {
+  filterElements.each((index, filter) => {
+    const filters = ($(filter).text() || '').trim().toLowerCase().split(/\s+/)
+    let hasSkills = false
```

Comment:
this whole thing with `hasSkills` is doing essentially what a [Array#some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) would do. 

you could use `.some` like this:

```js
const skills = $('.skills-list .skills-list-list > li').map(n => $(n).text() || '')
$('.skills-list-filter').each((index, filterNode) => {
  const filter = $(filterNode).text()
  const hasSkills = skills.some(skill => skill.inclides(filter))
  if (!hasSkills) $(filterNode).hide()
})
```